### ISSUE TITLE
Supported PC98 Games?

### ISSUE THREAD

**Issue by skyksit** [nbarkhina/DosWasmX#12]
I wonder if PC98 game mode is also supported.

When I tested it, it seems that PC98 games are not supported.

**Comment by nbarkhina** [comment]
I don't believe it is - while I was porting to WASM I vaguely remember ripping out a bunch of that code to get it to compile. I'll put it on my TODO list to try and get it back in.

**Comment by PoohBearIRC** [comment]
Seconding this request. PC-98 games have become more popular in recent years, but they still aren't super accessible to people who don't know where to find the game files, translation patches, or emulators capable of running them correctly. Being able to just run those games in a browser would be huge I think.
